plugins/Actions/ArchivingHelper.php

```diff
@@ -314,6 +315,7 @@ public static function reloadConfig()
         } else {
             self::$actionUrlCategoryDelimiter = self::$actionTitleCategoryDelimiter = $actionDelimiter;
         }
+        self::$actionTitleCategoryReverse = Config::getInstance()->General['action_title_category_reverse'];
```

Comment:
I agree, that this option should be moved to settings. But currently the options for $actionUrlCategoryDelimiter and $actionTitelCategoryDelimiter are also set system wide for all websites. I suggest to apply my patch now, and move all the relating options to website settings later.